### DISCUSSION TITLE
Error Message in Helper after Login

### DISCUSSION THREAD

**Discussion by Dragor17** [mainIine/foe-helfer-extension#3062] [General]
![Bildschirmfoto 2024-10-06 um 13 09 01](https://github.com/user-attachments/assets/366051db-451d-41b5-9819-e134bdb12234)
Please delete my Account, so i can Create a New one, the old Account don´t Work anymore, see Message in Attachement
My ID: 849210503

Thank you.

**Comment by sputnik-wulf** [comment]
I have the same problem. My ID is 849945049-
My name ist sputnik-wulf.
Additionally , whwn I chance from world Norasil de13 to Eldenborough de5 then apears another name called "Consurator Simoff". I not understand this.
When I will open the market place for LB sets, then appears a similar error warning  to Dragor17.
Therfore pleace delete my Account, that I have the chance to restart and may be you could explain me the problem.

**Comment by Bernini1** [comment]
Greetings, I am also having the same problem.   ID 11245188
In game name is Bernini and I cannot log in to FOEHelper site for city planner...which is the best I have seen anywhere.
Please, can this problem be fixed? Can deleting my account help?
Thank you :)

**Comment by xfranz1** [comment]
Very similar problem here... any solution?

**Comment by Linfex12** [comment]
Hello, I am also having the same problem. ID 849004606
In game name is Linfex and I cannot log in to FOE_Helper site "city planner".
Please, can anybody help me? Or can deleting my account? I can not delete my account. See the error message below.
Warning: Undefined array key "player" in /usr/www/users/foecre/foe-helper.com/core/modul/user/pages/delete.php on line 6
